spaceship/views.py

```diff
@@ -706,6 +750,26 @@ def edit():
       else:
         return jsonify({'ok': True})
 
+  elif table_name == 'mission':
+    try:
+      mission = Mission.get(Mission.id == object_id)
+    except Mission.DoesNotExist:
+      return jsonify({'ok': False})
+    # TODO acl
```

Comment:
lol. watch out, i'm coming to edit your mission!

Reply:
nooo